benchmarks/benchmarks/stats.py

```diff
@@ -41,6 +42,26 @@ def time_barnard_exact(self, alternative):
         resBarnard = stats.barnard_exact(self.a, alternative=alternative)
 
 
+class Kendalltau(Benchmark):
+    param_names = ['nan_policy','method','variant']
+    params = [
+        ['propagate', 'raise', 'omit'],
+        ['auto', 'asymptotic', 'exact'],
+        ['b', 'c']
+    ]
+
+    def setup(self, nan_policy, method, variant):
+        a = [x for x in range(0, 200)]
+        random.shuffle(a)
+        b = [x for x in range(0, 200)]
+        random.shuffle(b)
```

Comment:
Might be a bit more concise and consistent with other benchmarks/testing to use `np.arange(200)` and `np.random.shuffle()` instead of the standard library version equivalents.

Either way, we'd typically pin the seed down to be consistent as well.